tests/vitest.config.ts

```diff
@@ -4,7 +4,7 @@ const ONE_SECOND = 1_000;
 
 export default defineConfig({
   test: {
-    testTimeout: 60 * ONE_SECOND,
+    testTimeout: 100 * ONE_SECOND,
```

Comment:
This snuck in with other changes from other PRs, but started revealing itself here -- some of our tests have gotten slower (the e2e tests), since they now build _and_ run tests all in the same test.